Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.cpp

```diff
@@ -1037,7 +1037,7 @@ void GStreamerRegistryScanner::fillAudioRtpCapabilities(Configuration configurat
     if (!m_audioRtpExtensions) {
         auto extensions = m_commonRtpExtensions;
         extensions.appendVector(m_allAudioRtpExtensions);
-        m_audioRtpExtensions = probeRtpExtensions(extensions);
+        m_audioRtpExtensions = probeRtpExtensions(WTFMove(extensions));
```

Comment:
Taking into account what you are doing above, `entensions` is going to be alive after calling `probeRtpExtensions` because the funcion is not assigning or moving that inside anywhere.